lib/extends/models/decidim/authorization_extends.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+module AuthorizationExtends
+  extend ActiveSupport::Concern
+
+  included do
+    after_commit :export_to_user_extended_data, if: proc { |authorization|
+      ENV["AUTO_EXPORT_AUTHORIZATIONS_DATA_TO_USER_DATA_ENABLED_FOR"].to_s.split(",").include?(authorization.name)
```

Comment:
Prefer to move env variable into Rails secrets and set a default value to prevent `to_s` on NilClass